### ISSUE TITLE
Prevent playlist auto-play not working

### ISSUE THREAD

**Issue by Wolf68k** [YePpHa/YouTubeCenter#2099]
I can see that it's set it to disable the Auto-play but the playlist still plays anyway.

**Comment by mb1234** [comment]
Yes this doesn't work. We're not allowed to mention another addon that works for this issue but is inferior in every other way. For what it's worth it's https://github.com/ParticleCore/Particle/. You need both installed to get any reasonable experience out of youtube (sadly).

**Comment by Wolf68k** [comment]
That didn't seem to fix it either.
It must be partly with Youtube and partly with my FF profile folder.
With that addon the only thing it does do is when flip that blue auto-play to off then load another video it stays off. But with YTC it's turned back on but looks like it's turned off. That addon doesn't even try to flip the switch to off even though I told the settings to do so.

I deleted all YT cookies I could find in CookieCuller and disabled both addons and the autoplay still resets to on. But with another browser it seems like it stays off, or I missed a step in my testing that made it seem that way; it's been a while so I forget.
